src/assets/json/checklist/tests-web-en.json

```diff
@@ -822,6 +822,20 @@
 	"wcag": ["1.3.5"],
 	"ID": "testWebID-55",
 	"IDorigin": "testWebID-55"
+}, {
+	"themes": "Forms",
+	"title": "Avoid redundant entries",
+	"type": ["Manual test"],
+	"tests": ["Identify processes that include filling out forms.", "Identify the fields requesting the same data."],
+	"verifier": ["The user does not have to enter the same information twice during the same process."],
```

Comment:
According to WCAG (https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry.html), it sould be 'in the same session' instead of 'in the same process'.

Reply:
both of them